react/rollup.config.ts

```diff
@@ -0,0 +1,58 @@
+import * as path from 'path'
+import resolve from '@rollup/plugin-node-resolve'
+import commonjs from '@rollup/plugin-commonjs'
+import typescript from 'rollup-plugin-typescript2'
+import json from '@rollup/plugin-json'
+import packageJson from './package.json'
+
+const plugins = [
+    json(),
```

Comment:
Please correct me if I'm wrong, but I don't believe this would serve a purpose as there are no .json files used within the package.

Reply:
Yeah, copied this from another project of mine. I will clean it up based on your feedback